includes/config.php

```diff
@@ -177,3 +177,25 @@ function fontawesome_version() {
 }
 
 add_filter( 'ACFFA_override_major_version', __NAMESPACE__ . '\fontawesome_version' );
+
+
+/**
+ * Overriding the parent theme function to add in feeds.
+ *
+ * @since 1.0.1
+ * @author RJ Bruneel
+ **/
+function ucfwp_kill_unused_templates() {
```

Comment:
Just to avoid confusion, it might make sense to remove the `ucfwp_` prefix from this new function.